packages/rum/test/specHelper.ts

```diff
@@ -174,6 +178,12 @@ export function setup(): TestSetupBuilder {
       })
       return setupBuilder
     },
+    withInternalContext() {
+      buildTasks.push(() => {
+        internalContext = startInternalContext(FAKE_APP_ID, session, parentContexts)
```

Comment:
What's the threshold between using `beforeBuild` and adding new domain-specific methods in the `specHelper`? I feel that if the function to test (here `startInternalContext`) is used in a single spec file, we shouldn't add it to the common `specHelper`.

Reply:
yes, that is not clear for now.
Here, I chose that way to avoid to add parentContexts to beforeBuild parameters.
I'd be in favor of having a look at that at the same time that we have a look at remaining rum.ts test, to see which behavior we want to keep.